### ISSUE TITLE
IFN/ENIT-database

### ISSUE THREAD

**Issue by mohamadnajiya** [najwanaamane/OCR-arabic#1]
How can I download the database to test and check?

**Comment by someone185** [comment]
hello,
i have put the link in the readme
________________________________
De : mohamadnajiya ***@***.***>
Envoyé : mardi 3 décembre 2024 11:34
À : najwanaamane/OCR-arabic ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [najwanaamane/OCR-arabic] IFN/ENIT-database (Issue #1)


How can I download the database to test and check?

**Comment by mohamadnajiya** [comment]
Oh apologies. I missed that thank you

**Comment by someone185** [comment]
No need to apologise, i had just added the link after you mentionned it.
thanks for reaching out and for your interest in my project.

________________________________
De : mohamadnajiya ***@***.***>
Envoyé : mardi 3 décembre 2024 20:59
À : najwanaamane/OCR-arabic ***@***.***>
Cc : Najwa NAAMANE ***@***.***>; Comment ***@***.***>
Objet : Re: [najwanaamane/OCR-arabic] IFN/ENIT-database (Issue #1)


Oh apologies. I missed that thank you

**Comment by mohamadnajiya** [comment]
Perfect. Can you please also share the requirements please. tensorflow version ,keras version ,ect...
Because I ran the same code and I got different results for the validation loss and getting all results for the predictions and Best-path decoding Incorrect word.

**Comment by someone185** [comment]
i added the requirements. You should run the test script to get correct predictions, the code in the main script doesn't apply decoding correctly so it might give incorrect words.  But that was rectified in test1000.ipynb. Hope this helps
